Backend/app/rider-platform/rider-app/Main/src/SharedLogic/MerchantConfig.hs

```diff
@@ -155,9 +131,7 @@ checkFraudDetected riderId merchantOperatingCityId factors merchantConfigs mSear
         MoreSearching -> do
           searchCount :: Int <- sum . catMaybes <$> SWC.getCurrentWindowValues (mkSearchCounterKey mc.id.getId riderId.getId) mc.fraudSearchCountWindow
           pure $ searchCount >= mc.fraudSearchCountThreshold
-        TotalRides -> do
-          totalRidesCount :: Maybe Int <- getTotalRidesCountAndUpdate riderId mSearchReq
-          pure $ maybe False (<= mc.fraudBookingTotalCountThreshold) totalRidesCount
+        TotalRides -> pure $ maybe False (<= mc.fraudBookingTotalCountThreshold) (mSearchReq >>= DSR.totalRidesCount)
```

Comment:
will this come as (Just 0) for every search and new customers ?

Reply:
for new it will come as 0 and for old it will be nothing